### PR TITLE
add app loading spinner (change spinner for table & time series graph)

### PR DESCRIPTION
**Pull request by nickyrong**
spinner now use the waiter{} package

### CODE REVIEW

**Comment by nickyrong** [comment]
**For the app loading spinner**, the spinner is initiated at the beginning of the ui.R
`waiter_show_on_load(html = loading_screen), # place at the top before content`

Also need to tell when the app loading completes, therefore need to put a spinner closing at the end of server.R
`waiter_hide()`

**For individual component spinner,** 
previously, the spinner wrapper is in the ui.R. The new package waiter{} does the wrapping in server.R

At the beginning of server.R, we need to initialize a spinner for each component that we want to spin
`  # Here set up any elements that require spinner`
` spin_tsgraph <- Waiter$new("tsgraph")`
` spin_table <- Waiter$new("table")`

Then in the rendering, place a spinner show function
`spin_table$show() #show spinner` for example

**Comment by WraySmith** [comment]
I don't believe this branch has the waiter package spinners? You can directly commit to this working branch and it will auto update this PR.

File: app/server.R [hunk 1]
```diff
@@ -11,10 +11,9 @@ library(lmom) # frequency distribution fitting
 library(plotly) # interactive plots
 library(rlang) # distribution error handling
 library(renv) # package version control
-library(shinybusy) # busy indicator for rendering plots/tables
 library(FlowScreen) # hydrograph and trend test functionality
 library(pastecs) # quick descriptive stats stat.desc()
-library(httr) #http_status() to check HYDAT version
```
Comment (WraySmith) [inline app/server.R:17]: isn't this still required to go and check the HYDAT version?

File: app/ui.R [hunk 2]
```diff
@@ -4,16 +4,31 @@ library(leaflet)
...
+# a notorious gif
+gif <- paste0("https://media1.tenor.com/images",
+              "/cb27704982766b4f02691ea975d9a259/tenor.gif?itemid=11365139")
+
+loading_screen <- tagList(
+  h3("Give me a sec......", style = "color:gray;"),
+  img(src = gif, height = "200px")
+)
+#--------------------------
+
```
Comment (WraySmith) [inline app/ui.R:15]: if this is that gif, it should be removed

File: app/server.R (hunk 1 above)
Comment (nickyrong) [reply app/server.R:17]: it is added back now

File: app/ui.R (hunk 2 above)
Comment (nickyrong) [reply app/ui.R:15]: sigh....k, going back to use the spinning circle instead of gif

**Review by nickyrong — COMMENTED** [review]
accepted all comments from Nate

**Review by WraySmith — APPROVED** [review]

**Pull request merged**